tl/controllers/middleware.rst

```diff
@@ -51,17 +51,17 @@ Ang CakePHP ay nagbibigay ng ilang mga middleware para i-handle ang karaniwang m
 
 .. _using-middleware:
 
-Using Middleware
+Paggamit sa Middleware
 ================
 
-Middleware can be applied to your application globally, or to individual
-routing scopes.
+Ang Middleware ay pwedeng i-apply sa iyong aplikasyon na pangkalahatan, o para indibidwal
```

Comment:
- change i-apply to gamitin
- para sa indibidwal

Reply:
done change